Model/Enums/RequestType.cs

```diff
@@ -0,0 +1,12 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace Model.Enums
+{
+    public enum RequestType
+    {
+        Artist = 1,
+        Song = 2
```

Comment:
`Song = 2` kan gewoon `Song,` worden.
Enums hebben al auto-increment. Dus als je niet op 0 start (maar op 1 in dit geval), hoef je alleen maar het startpunt in te stellen.

Reply:
Prima

Reply:
ty